### PR TITLE
Allow to use authorization in clickhouse

### PR DESCRIPTION
**Pull request by turbotankist**
Clickhouse need to be protected by auth,
So there is a PR with it.
Clickhouse has default user "default" with empty ("") password

### CODE REVIEW

File: sentry/templates/_helper.tpl [hunk 1]
```diff
@@ -258,6 +258,53 @@ Set ClickHouse port
 {{- end -}}
 {{- end -}}
 
+{{/*
+Set ClickHouse Databse
```
Comment (J0sh0nat0r) [inline sentry/templates/_helper.tpl]: nit: database

File: sentry/values.yaml [hunk 2]
```diff
@@ -406,6 +406,18 @@ clickhouse:
...
             enabled: false
+      users:
+        enabled: false
+        user:
+        # the first user will be used if enabled
+        - name: default
+          config:
+            password: ""
+            networks:
+            - ::/0
```
Comment (J0sh0nat0r) [inline sentry/values.yaml:417]: Should we include `0.0.0.0/0` as a default here? The IPv6 rule didn't seem sufficient to allow access when testing locally. (not that users shouldn't put their cluster subnets here anyway, maybe worthy of a comment?)

**Review by J0sh0nat0r — APPROVED** [review]
LGTM 👍 Thanks for the PR!
Terribly sorry it took me this long to get around to reviewing this.

File: sentry/values.yaml (hunk 2 above)
Comment (turbotankist) [reply sentry/values.yaml:417]: I just copied default values from 
https://github.com/sentry-kubernetes/charts/blob/develop/clickhouse/values.yaml#L324
Also ::/0 usually mean both IPv4 and IPv6 subnetworks
https://clickhouse.tech/docs/en/operations/settings/settings-users/  
```
To open access for user from any network, specify:
<ip>::/0</ip>
```

File: sentry/templates/_helper.tpl (hunk 1 above)
Comment (turbotankist) [reply sentry/templates/_helper.tpl]: fixed

**Comment by Mokto** [comment]
Thanks @J0sh0nat0r @turbotankist

**Pull request merged**